src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -7020,6 +7056,53 @@ private void ApplyMemberPostConditions(BoundExpression? receiverOpt, MethodSymbo
             ApplyMemberPostConditions(receiverSlot, method);
         }
 
+        // For an instance method, or a non-static local function in an instance method, returns the slot for the `this` parameter
+        // For a static method, or a static local function in a static method, returns 0
```

Comment:
>For a static method, or a static local function in a static method [](http://example.com/codeflow?start=11&length=66)

Should this also include non-static local function in a static method? If so, perhaps: "For a static method, or a local function in a static method, or a static local function ...".